src/pages/CatEdit.js

```diff
@@ -72,6 +73,7 @@ class CatEdit extends Component {
                 />
               </FormGroup>
               <Button name="submit" onClick={this.handleSubmit}> Update Cat! </Button>
+              <p><NavLink to={`/`}><Button>Go Back to Homepage</Button></NavLink></p>
```

Comment:
Since you have three JSX tags here, you might consider dropping them onto new lines to show the nesting format. 
The `to={`/`}` isn't wrong but since you are just passing a `/` it feels a little excessive to wrap curlies just to use backticks without needing string interpolation. MIght consider `to="/"`